packages/utils/test/abi_encoder/return_values_test.ts

```diff
@@ -0,0 +1,78 @@
+import * as chai from 'chai';
+import 'mocha';
+
+import { AbiEncoder } from '../../src/';
+import { chaiSetup } from '../utils/chai_setup';
+
+import * as ReturnValueAbis from './abi_samples/return_value_abis';
+
+chaiSetup.configure();
+const expect = chai.expect;
+
+describe('ABI Encoder: Return Value Encoding/Decoding', () => {
+    it('No Return Value', async () => {
+        // Decode return value
+        const method = new AbiEncoder.Method(ReturnValueAbis.noReturnValues);
+        const returnValue = '0x';
+        const decodedReturnValue = method.decodeReturnValues(returnValue);
+        const expectedDecodedReturnValue: any[] = [];
+        const decodedReturnValueJson = JSON.stringify(decodedReturnValue);
```

Comment:
Instead of stringifying these, we should use:

```typescript
expect(decodedReturnValue).to.deep.equal(expectedDecodedReturnValue);
```

`JSON.stringify` ordering is not guaranteed, so these tests could all technically fail.